portality/formcontext/formcontext.py

```diff
@@ -1885,6 +1884,11 @@ def __init__(self, source, form_data, user):
     def set_template(self):
         self.template = "publisher/metadata.html"
 
+    def render_template(self, **kwargs):
+        self._check_for_author_errors()
```

Comment:
you're not calling `_check_for_author_errors()` with kwargs, so will it find anything?

Reply:
Good point.